ui/raidboss/timeline.ts

```diff
@@ -883,7 +883,7 @@ export class TimelineUI {
 
     this.timerlist = document.getElementById('timeline');
     if (this.timerlist)
-      this.timerlist.style.gridTemplateRows = `repeat(${this.options.MaxNumberOfTimerBars}, 1fr)`;
+      this.timerlist.style.gridTemplateRows = `repeat(${this.options.MaxNumberOfTimerBars}, min-content)`;
```

Comment:
Was this the cause of the truncating you were looking at?

Reply:
Since grid rows and grid columns can't be used as transition fields, you have to use width, min/max-width, height, or min/max-height as the transition field for the elements in the grid. With this as 1fr, changing the height on the top element (as the animation for collapsing the bar) won't affect the other bars, and it will simply do the animation and then "blink" the 2nd bar into the top bar slot (as is currently happening). By setting this to min-content, the grid rows are more flexible and will take on changes to height. By changing the removed bar's height, the other bars will "slide" into position.